client/game.hpp

```diff
@@ -56,8 +56,8 @@ namespace tec {
 		GameStateQueue game_state_queue;
 		ServerConnection server_connection;
 		RenderSystem rs;
-		VComputerSystem& vcs;
 		PhysicsSystem& ps;
+		VComputerSystem& vcs;
```

Comment:
This re-order is because of constructor initialization order vs class declaration order.